app/controllers/oral_history_access_requests_controller.rb

```diff
@@ -0,0 +1,49 @@
+# PUBLIC FACING
+# Staff-facing actions are in app/controllers/admin/oral_history_access_requests_controller.rb
+class OralHistoryAccessRequestsController < ApplicationController
+
+  before_action :set_work
+
+  # GET /works/4j03d097t/request_oral_history_access
+  def new
+    @oral_history_access_request = cls.new
+  end
+
+  # POST "/request_oral_history_access"
+  def create
+    @oral_history_access_request = cls.new(oral_history_access_request_params)
+    @oral_history_access_request.work = @work
+    if @oral_history_access_request.save
+      redirect_to work_path(@work.friendlier_id), notice: 'Your request has been logged.'
```

Comment:
For now, since this functionality isn't actually done yet, let's actually leave it in a state where nobody can accidentally think it's working if we accidentally turn it on in production. Instead of the redirect with a message that tells them it's been logged, how about:

    render plain: "This functionality is not activated yet, and is only present for testing!"

Reply:
Sure, I can see that. Will do.